datalad/support/annexrepo.py

```diff
@@ -1154,7 +1152,7 @@ def info(self, files, batch=False, fast=False):
             if not j['success']:
                 j = None
             else:
-                assert(j.pop('success') == True)
+                assert(j.pop('success') is True)
```

Comment:
have fun:
```
$> python -c 'True = 0; print(bool(True))'
False
```
I think I saw it redefined by some serializations somewhere... evil...  fixed in python3 though